src/redux/features/books/bookSlice.js

```diff
@@ -0,0 +1,40 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  books: [{
+    item_id: 'item1',
+    title: 'The Great Gatsby',
+    author: 'John Smith',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item2',
+    title: 'Anna Karenina',
+    author: 'Leo Tolstoy',
+    category: 'Fiction',
+  },
+  {
+    item_id: 'item3',
+    title: 'The Selfish Gene',
+    author: 'Richard Dawkins',
+    category: 'Nonfiction',
+  }],
+};
+const bookSlice = createSlice({
+  name: 'books',
+  initialState,
+  reducers: {
+    removeBooks: (state, action) => {
+      const id = action.payload;
+      const filtered = state.books.filter((book) => book.item_id !== id);
+      return { ...state, books: filtered };
+    },
```

Comment:
- [ ] Great job so far. However, there's littile issue here. I was able to delete all the populated book objects without any issue. But after adding new books, I was unable to remove them one after the other, but remove all the added books at once when intending to remove only just one book. Kindly look at it and fix.

### Your App Behavior
[app-behave.webm](https://user-images.githubusercontent.com/26597432/230059307-7afd0f89-1be3-4cf2-916e-deffe422f9f0.webm)

- You may consider checking this approach :point_down: 
```
  removeBook: (state, action) => {
      state.books = state.books.filter((book) => book.id !== action.id)
    },

```